scripts/reorder_json_keys.py

```diff
@@ -1,22 +1,37 @@
 import json
 import os
 
-from recipe_scrapers._utils import get_abstract_methods
+from recipe_scrapers._utils import get_abstract_methods, get_nutrition_keys
 
 KEYS = get_abstract_methods()
+NUTRITION_KEYS = get_nutrition_keys()
 
 
 def reorder_json_keys(file_path):
     with open(file_path, encoding="utf-8") as file:
         data = json.load(file)
 
     reordered_data = {key: data[key] for key in KEYS if key in data}
-
     if list(data.keys()) != list(reordered_data.keys()):
-        with open(file_path, "w", encoding="utf-8") as file:
-            print(f"Re-ordering JSON keys for: {file_path}")
-            json.dump(reordered_data, file, indent=2, ensure_ascii=False)
-            file.write("\n")
+        data = reordered_data
```

Comment:
This line is important but a little bit subtle.  I wonder whether it might be clearer _not_ to change what the `data` variable points to, but instead to perform the `nutrients` ordering check on the `reordered_data`.  What do you reckon?

Reply:
I implemented a similar change, let me know what you think